Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Pointers/PokePointer.cs

```diff
@@ -17,6 +17,8 @@ public class PokePointer : BaseControllerPointer, IMixedRealityNearPointer
         private float closestDistance = 0.0f;
 
         private Vector3 closestNormal = Vector3.forward;
+        // previous frame pointer position
+        public Vector3 PreviousPosition { get; private set; } = Vector3.zero;
```

Comment:
not sure if i should move that into the IMixedRealityNearPointer interface?

Reply:
I think it's fine like this. If you put it in the interface then we'd have to rely on every pointer implementing this correctly.